tests/testthat/setup.R

```diff
@@ -19,3 +19,19 @@ fast_print <- function(x) {
     cat(toString(x[[pg_num]]))
   })
 }
+
+
+compare_paginations <- function(paglst1, paglst2) {
+  if (length(paglst1) != length(paglst2))
+    return(FALSE)
```

Comment:
this should be in the expect

Reply:
You're right, at first I was going to call compare_paginations within an expect and didn't go back and fix this. Good catch, thanks

Reply:
Fixed in latest commit in PR